sdk/formrecognizer/Azure.AI.FormRecognizer/src/CustomFormModelInfo.cs

```diff
@@ -12,26 +12,40 @@ namespace Azure.AI.FormRecognizer.Training
     [CodeGenModel("ModelInfo")]
     public partial class CustomFormModelInfo
     {
-        /// <summary>
-        /// Initializes a new instance of the <see cref="CustomFormModelInfo"/> class.
-        /// </summary>
-        /// <param name="modelId">The unique identifier of the model.</param>
-        /// <param name="trainingStartedOn">The date and time (UTC) when model training was started.</param>
-        /// <param name="trainingCompletedOn">The date and time (UTC) when model training completed.</param>
-        /// <param name="status">The status of the model.</param>
-        internal CustomFormModelInfo(string modelId, DateTimeOffset trainingStartedOn, DateTimeOffset trainingCompletedOn, CustomFormModelStatus status)
+        /// <summary> Initializes a new instance of CustomFormModelInfo. </summary>
+        /// <param name="modelId"> Model identifier. </param>
+        /// <param name="status"> Status of the model. </param>
+        /// <param name="trainingStartedOn"> Date and time (UTC) when the model was created. </param>
+        /// <param name="trainingCompletedOn"> Date and time (UTC) when the status was last updated. </param>
+        /// <param name="displayName"> Optional user defined model name (max length: 1024). </param>
+        /// <param name="properties">Model properties, like for example, if a model is composed.</param>
+        internal CustomFormModelInfo(string modelId, CustomFormModelStatus status, DateTimeOffset trainingStartedOn, DateTimeOffset trainingCompletedOn, string displayName, CustomFormModelProperties properties)
         {
             ModelId = modelId;
+            Status = status;
             TrainingStartedOn = trainingStartedOn;
             TrainingCompletedOn = trainingCompletedOn;
-            Status = status;
+            DisplayName = displayName;
+            Properties = properties ?? new CustomFormModelProperties();
         }
 
         /// <summary>
         /// The unique identifier of the model.
         /// </summary>
         public string ModelId { get; }
 
+        /// <summary>
+        /// The optional display name of the model.
+        /// </summary>
+        [CodeGenMember("ModelName")]
+        public string DisplayName { get; }
+
+        /// <summary>
+        /// Model properties, like for example, if a model is composed.
+        /// </summary>
+        [CodeGenMember("Attributes")]
+        public CustomFormModelProperties Properties { get; }
```

Comment:
It would be too generic to call `Properties`. How about `customFormModelProperties`?

Reply:
The class is already called `CustomFormModelInfo`. to me it sounds redundant to have in the name `CustomFormModel`

Reply:
I will leave it as is for now, and we can keep the conversation with other languages open